Week5/solution week-5/week5.py

```diff
@@ -0,0 +1,29 @@
+Attribute=['sid','sname','city','contect_no']
+records=[[1,'sunny patel','vyara',9856432456],
+         [2,'rohit sharma','bajipura',9964789098],
+         [3,'rahul soni','surat',6543789090],
+         [4,'gaurav aahire','bardoli',8469546134],
+         [5,'malav patel','bharuch',9312477895]]
+list=[]

```

Comment:
Without taking empty list how can i do this code in another way?

Reply:
we can take input directly and output will be gives a list...